docs/includes/installation.txt

```diff
@@ -6,7 +6,15 @@ or from source.
 
 To install using `pip`,::
 
-    $ pip install -U django-celery-beat
+    $ pip install --upgrade django-celery-beat
+
+Installing the current default branch
+-------------------------------------
+
+    $ python3 -m venv .venv
```

Comment:
Same here: https://github.com/celery/django-celery-beat/pull/737#discussion_r1503724648 With these changes, it looks like the steps are more about venvs, not the actual goal of the doc.